src/TensorBoardLogger.jl

```diff
@@ -49,6 +51,7 @@ include("utils.jl")  # CRC Utils
 include("event.jl")
 include("Loggers/base.jl")
 include("Loggers/LogValue.jl")
+include("Loggers/LogCustomScalar.jl")
```

Comment:
Can you move the include down? in a separate block? I'd like to remind myself this is a plugin
```Julia
# Custom Scalar Plugin
include("Loggers/LogCustom...)
```